src/main/java/com/ericsson/ei/handlers/EventToObjectMapHandler.java

```diff
@@ -95,15 +95,14 @@ public void updateEventToObjectMapInMemoryDB(RulesObject rulesObject, String eve
             ArrayNode jsonNode = mapper.convertValue(list, ArrayNode.class);
             ((ObjectNode) entry).set(listPropertyName, mapper.readTree(jsonNode.toString()));
             String mapStr = entry.toString();
-            log.debug("MongoDbHandler Insert/Update Event: " + mapStr +
-            		"\nto database: " + databaseName + " and to Collection: " + collectionName);
+            LOGGER.debug("MongoDbHandler Insert/Update Event: {}\nto database: {} and to Collection: {}", mapStr, databaseName, collectionName);
```

Comment:
what is mapStr? Please rename variable.

Reply:
I may leave this for another story/PR. I'm not sure how to rename it.